Sources/ApolloCodegenLib/Templates/TemplateRenderer.swift

```diff
@@ -141,20 +143,99 @@ extension TemplateRenderer {
     """
     ).description
   }
+}
+
+// MARK: Extension - Access modifier
+
+fileprivate extension ApolloCodegenConfiguration.AccessModifier {
+  var swiftString: String {
+    switch self {
+    case .public: return "public " // there should be no spaces in these strings
+    case .internal: return ""
+    }
+  }
+}
+
+enum AccessControlModifierTypeDefinition {
```

Comment:
I'm open to renaming suggestions for these cases. They represent this structure:
```
<namespace> {
  <parent> {
    <member>
    <member>
  }
}
```
When there is a `namespace` (`enum` and it's `extension`) that is `public` the `parent` cannot have the `public` modifier whereas the `member` should all get the `public` modifier.

Reply:
Honestly I think these cases make sense, `namespace` and `member` especially I think reflect what they represent well, `parent` would be the only one that could _maybe_ change but I'm struggling to think of what would maybe better represent that.

Reply:
Change `AccessControlModifierTypeDefinition` to `AccessControlScope`